Secure data

NeferHikari opened the pull request:
Sprawdź proszę czy nic mi nie znikło i czy czegoś jeszcze nie zabrakło

NeferHikari committed:
Add some validation for data

NeferHikari committed:
Delete ERD.jpg

NeferHikari committed:
Delete ERD.png

NeferHikari committed:
Delete ERD.drawio

NeferHikari committed:
Add updated diagrams

NeferHikari committed:
Add check for EventStart and EventEnd

olszewskanatalia commented:
Takie są ENUM na backendzie:
BLIK, PayPal, Card, SMS, BankTransfer <= PaymentMethod
NotStarted, Success, Failure <= PaymentStatus
Created, InProgress, Paused, Ended <= EventState

olszewskanatalia committed:
Update enums

BLIK, PayPal, 'Card', SMS, BankTransfer <= PaymentMethod
NotStarted, Success, Failure <= PaymentStatus
Created, InProgress, Paused, Ended <= EventState

olszewskanatalia commented:
Dodałam poprawkę możesz sprawdzić

NeferHikari commented:
![obraz_2023-12-02_131358879](https://github.com/PO-2023Z/larpex-db/assets/107104083/5277ef80-96c5-4544-b6b8-fc35830fdc34)
 ogólnie to chyba not resolved chyba że BEnd coś zmienił

NeferHikari commented:
![image](https://github.com/PO-2023Z/larpex-db/assets/107104083/5a4df0c6-f65a-4061-89bd-36e9880820d6)
i w kodzie też jest

NeferHikari committed:
Update init.sql paymentState

olszewskanatalia commented:
A to widocznie zmienili to od czasu gdy nam wysyłali

olszewskanatalia commented:
To reszta gitara
